microsoftgraph/microsoft-graph-explorer-v4
# Fix: Return focus to feedback button

## Overview
Closes #1859 

### Demo

Optional. Screenshots, `curl` examples, etc.
![image](https://user-images.githubusercontent.com/45680252/179740072-2b513ebd-0d75-4db8-aa5a-5903f473dbd3.png)

After hitting Escape: 
![image](https://user-images.githubusercontent.com/45680252/179740120-f0faecca-c6aa-4d5d-af67-6a9bc7aba722.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Tab through the local instance upto the Feedback button
Hit Enter on your keyboard to launch the feedback modal
Hit 'Escape' on your keyboard to dismiss the modal
Notice that focus is restored to the Feedback button
